### PR TITLE
PHPCS resolve in main controllers templates

### PR DESCRIPTION
**Pull request by vaishaliagola27**
Related issue #1147

### CODE REVIEW

File: app/main/controllers/template/rtmedia-actions.php [hunk 1]
```diff
@@ -124,13 +131,15 @@ function rtmedia_add_album_selection_field( $media_type ) {
 			<?php
 			if ( isset( $rtmedia_query->query['context'] ) && 'group' === $rtmedia_query->query['context'] ) {
 				// show group album list.
-				$album_list = rtmedia_group_album_list( $selected_album_id = $curr_album_id );
+				$selected_album_id = $curr_album_id;
```
Comment (thrijith) [inline app/main/controllers/template/rtmedia-actions.php]: Do we need this assignment? Could we directly use `$curr_album_id` and remove usage of `$selected_album_id`?

File: app/main/controllers/template/rtmedia-actions.php [hunk 2]
```diff
@@ -124,13 +131,15 @@ function rtmedia_add_album_selection_field( $media_type ) {
...
+				$album_list        = rtmedia_group_album_list( $selected_album_id );
 			} else {
 				// show profile album list.
-				$album_list = rtmedia_user_album_list( $get_all = false, $selected_album_id = $curr_album_id );
+				$selected_album_id = $curr_album_id;
```
Comment (thrijith) [inline app/main/controllers/template/rtmedia-actions.php]: Same as above.

File: app/main/controllers/template/rtmedia-actions.php [hunk 3]
```diff
@@ -359,7 +369,7 @@ function rtmedia_link_in_footer() {
 
 	if ( $link ) {
 
-		$aff_id = ( '' != $option['rtmedia_affiliate_id'] ) ? '&ref=' . $option['rtmedia_affiliate_id'] : '';
+		$aff_id = ( '' !== $option['rtmedia_affiliate_id'] ) ? '&ref=' . $option['rtmedia_affiliate_id'] : '';
```
Comment (thrijith) [inline app/main/controllers/template/rtmedia-actions.php]: The option was removed in https://github.com/rtMediaWP/rtMedia/commit/c53c1fa5a4e741d38840d1dfe4fa17f9910a547d, we can remove this now.

File: app/main/controllers/template/rtmedia-actions.php (hunk 1 above)
Comment (vaishaliagola27) [reply app/main/controllers/template/rtmedia-actions.php]: yes. Doing it.

File: app/main/controllers/template/rtmedia-actions.php (hunk 3 above)
Comment (vaishaliagola27) [reply app/main/controllers/template/rtmedia-actions.php]: okay

**Pull request merged**